src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -11,31 +12,34 @@ interface Props {
 
 export const SuggestionItem: SFC<Props> = ({ href, display, label, query }) => {
   if (label === "FirstItem") {
+    // `color="black100"` is misleading.
+    //  The actual color doesn't really matter - the child element controls the displayed color.
+    //  We specify color only because it makes the text not underlined on hover.
```

Comment:
This seems like it would make a good API improvement PR

Reply:
Yeah, definitely. We've been talking about this quirk in the `<Link>` component, and [trying to figure out what to do with it](https://github.com/artsy/palette/issues/129#issuecomment-463686937). It meets the specs that were originally created in Zeplin, but it's confusing & misleading.